src/components/Image/src/Image.vue

```diff
@@ -220,6 +220,7 @@ export default {
 }
 
 .Image {
+	display: block;
```

Comment:
can you comment explaining this change? i don't understand how it's relevant to the rest of the PR

Reply:
cool, thanks for explaining